source/module_elecstate/module_dm/test/test_dm_R_init.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
+#define private public
```

Comment:
I have to say due to present program structure, this is a rather rude way to get access to private member who seems should have a `getter` function or something should be tested in another way but, this indicates there should be a re-design on the class in which those mistakenly set as private variables (and even functions) and, certainly a refactor, although it may be not of high priority.

Reply:
I understand this. However, the get function of `_DMR_save` will never be called in main code. I do not think define a get-function only for UnitTest is a good idea.

Reply:
> I have to say due to present program structure, this is a rather rude way to get access to private member who seems should have a `getter` function or something should be tested in another way but, this indicates there should be a re-design on the class in which those mistakenly set as private variables (and even functions) and, certainly a refactor, although it may be not of high priority.

I understand this. However, the get function of `_DMR_save` will never be called in main code. I do not think define a get-function only for UnitTest is a good idea.